### ISSUE TITLE
Workspace Manager Vertical Panel Support

### ISSUE THREAD

**Issue by naphipps** [adobe/brackets#12199]
Hi Brackets Developers!

Currently, from what I understand, there is no support for Vertical Panels in the ```Workspace Manager```.

Many extensions suffer from not being able to take advantage of Vertical Panels. (Like [gruehle's markdown preview](https://github.com/gruehle/MarkdownPreview) extension.)

Is there any suggestions you might have to accommodate for that or are we stuck with Horizontal Panels in the ```Workspace Manager``` for the time being?

**Comment by one-github** [comment]
+1

**Comment by MiguelCastillo** [comment]
Yeah - I really can't think of a way to do this with native brackets constructs.  I might be possible to do as an extension, but that requires messing with the DOM which might be error prone.  Have you entertained the idea of making a PR for this? :)

**Comment by one-github** [comment]
It's a pain for many extensions, just take the Git extension. It's just space the developer lacks above as the Git panel grows vertically with its file list.

**Comment by naphipps** [comment]
Hey @MiguelCastillo,

I have thought about it (that is, making a PR). Sadly, I don't have time at the moment, and I won't for another year or so.

(I just end up running two Brackets instances to deal with this in the meantime. It's annoying, but for projects I rely on brackets for, it works.)

(I wouldn't make an extension to do this either. That's just more overhead and rather sloppy. This means other extensions would then depend on this extension, etc and etc. That's not good for anyone - even if it works for the time-being. A PR would be the way to go.)
